topology/yaml_topology_schema.md

```diff
@@ -10,46 +10,59 @@ format below. We use an intuitive, loose format. The multiplicity is presented a
+                                                in dotted notation, e.g. 224.0.0.2>  {5}
                 {?}      rx_lie_v6_mcast_address: <unique v6 multicast address, e.g. FF02::0:2> (5)
-                {1}      rx_lie_port: <node-wide UDP Port used to receive LIEs> (4)
+                {1}      rx_lie_port: <node-wide UDP Port used to receive LIEs> {4}
                 {?}X     state_thrift_services_port: <TCP port to run state services>
                 {?}X     config_thrift_services_port: <TCP port to run config services>
                 {?}      generate_defaults: <boolean indicating whether southbound defaults are 
                                              generated, default is true>
-                {?}      active_key: <24-bit key number>
-                {?}      tie_validation: [none|permissive|loose|strict]  (6)
+                {?}      authentication_keys: <set of 24-bit key numbers>  {9}                             
+                {?}      tie_origination_authentication_key: <24-bit key number>
+                {?}      tie_authentication_validation: [none|permissive|loose|strict]        {6}
                 {1}      interfaces:
                 {*}         - name: <interface name string>
                 {?}           bandwidth: <in megabit units, if not given, schema default is used>
                 {?}           metric: <positive numerical metric > 0, if not given, schema default>
                 {?}           tx_lie_port: <UDP Port used to send LIEs, 
                                             must match remote node's rx_lie_port and be 
-                                            unique within the configuration> (3)
+                                            unique within the configuration> {3}
                 {?}           rx_lie_port: <UDP Port used to receive LIEs, 
                                             must match remote node's tx_lie_port and be 
-                                            unique within the configuration> (3)
+                                            unique within the configuration> {3}
                 {?}           rx_tie_port: <UDP Port used to receive TIEs, 
                                             automatically carried on LIEs, must be unique 
-                                            within the configuration> (3)
+                                            within the configuration>        {3}
                 {?}           advertise_subnet: <boolean indicating whether the interface subnet
                                                  is advertised in RIFT northbound, default is false>
-                {?}           active_key: <8-bit key number> 
-                {?}           accept_keys: <set of 8-bit key number>
-                {?}           link_validation: [none|permissive|loose|strict]  (6)       
+                {?}           active_authentication_key: <8-bit key number> 
```

Comment:
I see that you changed the documentation file (md file) for the topology schema. Perhaps I am looking in the wrong place, but as far as I can tell there are no corresponding code changes in any .py file to actually implement these documented changes. ( I selected "Changes from all commits" when reviewing this pull request, but I don't see any py file related to these documented changes. )

Reply:
hmm, that stuff been here since quite a while and there is code reading that and processing in e.g. config.py etc so not sure what the problem is. interoip tests all pass BTW as well.